java/org/contikios/cooja/plugins/Visualizer.java

```diff
@@ -641,24 +639,23 @@ public void mouseReleased(MouseEvent e) {
                   selectedMotes.add(cursorMote);
                 }
               }
-              break;
-            case DEFAULT_PRESS:
+            }
+            case DEFAULT_PRESS -> {
               if (cursorMote == null) { // Click on free canvas deselects all motes.
                 selectedMotes.clear();
               } else {                 // Click on mote selects single mote.
                 selectedMotes.clear();
                 selectedMotes.add(cursorMote);
               }
-              break;
-            case MOVING:
+            }
+            case MOVING ->
               // Release stops moving.
-              canvas.setCursor(Cursor.getDefaultCursor());
-              break;
-            case SELECTING:
+                    canvas.setCursor(Cursor.getDefaultCursor());
```

Comment:
Indentation looks strange here, at least when it's in combination with the comment.